tools/autoformat

```diff
@@ -30,6 +47,17 @@ process_file() {
 		printf "%u/%u\r" $(( ++COUNTER )) "$TOTAL"
 	fi
 }
-
 export -f process_file
-parallel process_file ::: "$FILES"
+
+if [[ "$NO_PARALLEL" = true ]] ; then
+	echo "Using non-parallel batch processing mode."
+	for fn in $FILES; do process_file "$fn"; done
+
+elif command -v parallel &> /dev/null; then
```

Comment:
Check for this right after getopt. You want to validate the expected environment before anything of substance is done.

Reply:
alright